CMakeLists.txt

```diff
@@ -255,6 +397,15 @@ configure_file(tools/notcurses.pc.in
   @ONLY
 )
 
+configure_file(tools/notcurses++.pc.in
```

Comment:
we probably want CMake support, too. you can cargo cult it off what I've already got.

I think we might want to combine the C/C++ into a single CMake module, and keep it as two pkg-config modules (as you already have it). Need to look into that, but I think that's the idiom I've seen.

Reply:
As long as the cmake module won't force anyone to link both libraries, sure, that's a good way to do it.

Reply:
It shouldn't so long as it's done correctly.